install-dev/langs/de/data/tab.xml

```diff
@@ -54,37 +54,37 @@
   <tab id="Languages" name="Sprachen"/>
   <tab id="Localization" name="Lokalisierung"/>
   <tab id="Localization_1" name="Lokalisierung"/>
-  <tab id="Locations" name="Locations" />
+  <tab id="Locations" name="Orte" />
   <tab id="Login" name="Anmeldung"/>
   <tab id="Logs" name="Log-Dateien"/>
   <tab id="Look_feel" name="Design" />
   <tab id="Maintenance" name="Wartung"/>
   <tab id="Manufacturers" name="Hersteller"/>
-  <tab id="Manufacturers_and_suppliers" name="Brands &amp; Suppliers" />
+  <tab id="Manufacturers_and_suppliers" name="Marken &amp; Lieferanten" />
   <tab id="Marketing" name="Marketing"/>
   <tab id="Menus" name="Menüpunkte"/>
   <tab id="Merchandise_Returns" name="Warenrücksendungen"/>
-  <tab id="Modules" name="Modules" />
-  <tab id="Modules_1" name="Modules" />
+  <tab id="Modules" name="Module" />
+  <tab id="Modules_1" name="Module" />
   <tab id="Monitoring" name="Kontrollübersicht"/>
   <tab id="Multistore" name="Multishop"/>
   <tab id="Order_Messages" name="Bestellnachrichten"/>
-  <tab id="Order_settings" name="Order Settings" />
-  <tab id="Order_settings_2" name="Order Settings" />
+  <tab id="Order_settings" name="Bestellungen" />
```

Comment:
Shouldn't we say "Bestellung Einstellungen" ?
Otherwise you'd have 2 pages with the same title in the menu. See line 74 to 76.